### DISCUSSION TITLE
Diff Window

### DISCUSSION THREAD

**Discussion by msscribner** [WinMerge/winmerge#2442] [General]
I have a new laptop and downloaded the latest WinMerge.  I found that in the DIFF window, I can't edit a file like I use to be able to with the the older version on my old laptop.   And instead of seeing a single merged file, the DIFF window is broken out into two.   Is there a way to enable the single window in the DIFF sections ?

**Comment by sdottaka** [comment]
By DIFF window, do you mean the Diff pane at the bottom as shown in the attached image? If so, you couldn't edit it in older versions either.

![image](https://github.com/user-attachments/assets/a12b38a9-dda2-4f15-9080-9a147c8d2e97)

> Is there a way to enable the single window in the DIFF sections?

Neither older nor the latest versions can display it like that. Were you using another application like the one below?

https://tortoisesvn.net/docs/release/TortoiseMerge_en/images/TMerge_OnePane.png
